tex/pgfcirctripoles.tex

```diff
@@ -865,151 +804,144 @@
 	}
 
 
-
+%%% start european logic ports (do really someone use them?) 
```

Comment:
As there are European users out there that actually use the European ports (like me) I would like to argue that the second half of the comment is unnecessary.

```suggestion
%%% start european logic ports
```